sds/src/match_validation/http_validator.rs

```diff
@@ -2,11 +2,11 @@ use super::{
     config::{CustomHttpConfig, RequestHeader},
     match_validator::MatchValidator,
 };
+use crate::match_validation::match_validator::RAYON_THREAD_POOL;
 use crate::{match_validation::config::HttpMethod, MatchStatus, RuleMatch};
 use crate::{scanner::RootCompiledRule, HttpValidatorOption};
 use ahash::AHashMap;
 use lazy_static::lazy_static;
-use rayon::prelude::*;
```

Comment:
Could/Do we have a way to enforce that `use rayon::prelude::*;` is never used globally?
This can probably be tackled as follow-up PR

Reply:
I'm not sure of a feasible way to enforce this. The global import itself doesn't actually change anything. The `RAYON_THREAD_POOL.install` is the important part. I just moved the imports to make it slightly harder to mess it up. Maybe other concurrency libraries make this easier.